Completed Flipclock

thehappypretzel opened the pull request:
Includes texture for the numbers and animation functionality with texture swapping with Blueprint instance.

Note: has changes to L_default to replace current flipclock with the blueprint version

thehappypretzel committed:
Completed Flipclock

Includes texture for the numbers and animation functionality with texture swapping with Blueprint instance

jegood reviewed (changes requested):
![update_loop](https://user-images.githubusercontent.com/9098464/53705767-c650a800-3df4-11e9-982e-30f3dc05d4b8.PNG)

If you're not going to use the TextRender component in non-debug mode, don't use it's SetText function. This will cause lots of unintended errors since you've previously told unreal to kill the component, hence it will try update something that doesn't exist!

You'll need to check this per tick and only update in its own SetText block if "Display Debug Text" is true.

thehappypretzel committed:
Fixed updating non-existant components

Fixed trying to update the debug text if it was deleted. Also cleaned up the blueprint and added more comments.

jegood reviewed (approved):
👍
